go_sequencer/common/zk.go

```diff
@@ -80,9 +81,9 @@ type ZKInputs struct {
 
 	// FromIdx
 	FromIdx []*big.Int `json:"fromIdx"` // uint64 (max nLevels bits), len: [maxTx]
-	// AuxFromIdx is the Idx of the new created account which is
+	// AuxFromAccountIdx is the AccountIdx of the new created account which is
 	// consequence of a L1CreateAccountTx
-	AuxFromIdx []*big.Int `json:"auxFromIdx"` // uint64 (max nLevels bits), len: [maxTx]
+	AuxFromAccountIdx []*big.Int `json:"auxFromIdx"` // uint64 (max nLevels bits), len: [maxTx]
 
 	// ToIdx
 	ToIdx []*big.Int `json:"toIdx"` // uint64 (max nLevels bits), len: [maxTx]
```

Comment:
Here, `FromIdx`, `AuxFromIdx`, and `ToIdx` are all related account, so I think if you want to change `AuxFromIdx` into `AuxFromAccountIdx`, you need to change `FromIdx` and `ToIdx`, too.

Reply:
Yupp, @kaiden-araki 
The reason of not changing FromIdx and ToIdx was to to leave maximum props as it is and once circom proover is completed then update the same.